Bookstack update fails and deletes everything

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Bookstack

### 📂 What was the exact command used to execute the script?

update

### 📝 Provide a clear and concise description of the issue.

When trying to update bookstack, it fails and installation is gone

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

just do update...

### ❌ Paste the full error output (if available).

  ✔️   Services Stopped
⠏ Updating Bookstack to v25.02.2unzip:  cannot find or open /opt/v25.02.2.zip, /opt/v25.02.2.zip.zip or /opt/v25.02.2.zip.ZIP.
⠋ Updating Bookstack to v25.02.2
[ERROR] in line 39: exit code 0: while executing command unzip -q /opt/v${RELEASE}.zip -d /opt

root@bookstack:~# ls -ahl /opt/
total 5.2M
drwxr-xr-x  3 root     root     4.0K Apr  2 23:27 .
drwxr-xr-x 18 root     root     4.0K Apr  2 23:26 ..
-rw-r--r--  1 root     root     5.2M Apr  2 23:27 BookStack-25.02.2.zip
-rw-r--r--  1 root     root        8 Mar 22 21:16 Bookstack_version.txt


----------------------------


Next update:
  ✖️   No Bookstack Installation Found!
  


### 🖼️ Additional context (optional).

The problem did not exist in this variant before this topic:
https://github.com/community-scripts/ProxmoxVE/issues/3557#issuecomment-2769076769

--- a/ct/bookstack.sh
+++ b/ct/bookstack.sh
@@ -36,7 +36,7 @@ function update_script() {
     msg_info "Updating ${APP} to v${RELEASE}"
     mv /opt/bookstack /opt/bookstack-backup
     curl -fsSL "https://github.com/BookStackApp/BookStack/archive/refs/tags/v${RELEASE}.zip" -o "/opt/BookStack-${RELEASE}.zip"
-    unzip -q /opt/BookStack-"${RELEASE}".zip -d /opt
+    unzip -q "/opt/BookStack-${RELEASE}.zip" -d /opt
     mv "/opt/BookStack-${RELEASE}" /opt/bookstack
     cp /opt/bookstack-backup/.env /opt/bookstack/.env
     cp -r /opt/bookstack-backup/public/uploads/* /opt/bookstack/public/uploads/ || true

--- a/ct/bookstack.sh
+++ b/ct/bookstack.sh
@@ -59,7 +59,7 @@ function update_script() {
 
     msg_info "Cleaning Up"
     rm -rf /opt/bookstack-backup
-    rm -rf "/opt/BookStack-${RELEASE}".zip"
+    rm -rf "/opt/BookStack-${RELEASE}.zip"
     msg_ok "Cleaned"
     msg_ok "Updated Successfully"
   else

--- a/ct/bookstack.sh
+++ b/ct/bookstack.sh
@@ -36,13 +36,13 @@ function update_script() {
     msg_info "Updating ${APP} to v${RELEASE}"
     mv /opt/bookstack /opt/bookstack-backup
     curl -fsSL "https://github.com/BookStackApp/BookStack/archive/refs/tags/v${RELEASE}.zip" -o "/opt/BookStack-${RELEASE}.zip"
-    unzip -q /opt/v${RELEASE}.zip -d /opt
-    mv /opt/BookStack-${RELEASE} /opt/bookstack
+    unzip -q /opt/BookStack-"${RELEASE}".zip -d /opt
+    mv "/opt/BookStack-${RELEASE}" /opt/bookstack
     cp /opt/bookstack-backup/.env /opt/bookstack/.env
     cp -r /opt/bookstack-backup/public/uploads/* /opt/bookstack/public/uploads/ || true
     cp -r /opt/bookstack-backup/storage/uploads/* /opt/bookstack/storage/uploads/ || true
     cp -r /opt/bookstack-backup/themes/* /opt/bookstack/themes/ || true
-    cd /opt/bookstack
+    cd /opt/bookstack || exit
     export COMPOSER_ALLOW_SUPERUSER=1
     $STD composer install --no-dev
     $STD php artisan migrate --force
@@ -59,7 +59,7 @@ function update_script() {
 
     msg_info "Cleaning Up"
     rm -rf /opt/bookstack-backup
-    rm -rf /opt/v${RELEASE}.zip
+    rm -rf "/opt/BookStack-${RELEASE}".zip"
     msg_ok "Cleaned"
     msg_ok "Updated Successfully"
   else

